packages/evm/vitest.config.browser.mts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from 'vitest/config'
+import { nodePolyfills } from 'vite-plugin-node-polyfills'
+import wasm from 'vite-plugin-wasm'
+
+const config = defineConfig({
```

Comment:
@roninjin10 do you have any insights on why an arbitrary browser test script seems to always fail on CI?  All of our browser tests pass locally but when they run on CI, we get these `failed to resolve dynamically imported module...` errors.  I've tried all manner of config settings trying to remove parallel test runs and isolated environments, etc, with no success.  At this point, I haven't come up with a reproducible "minimal" build to point the vitest devs at so have hesitated to open an issue there (though may do so soon)

Reply:
@acolytec3 try setting `VITEST_BROWSER_DEBUG=true` environment variable

Reply:
Btw if the debug logging doesn't provide any insight let me know I'd be happy to look at this

Reply:
It just keeps doing this sort of thing - https://github.com/ethereumjs/ethereumjs-monorepo/actions/runs/8943487337/job/24568309037#step:9:74

Debug logging doesn't seem to tell me anything more.  It's not always the `tx` test run that fails either.  I can comment out `tx` and another one will fail the same way (or I can tell vitest to skip the one test file that's not being loaded and it will still fail on `tx` with a different file having no tests fail).  Very weird.

Reply:
It seems to have started after vitest v1.5.1 was released

Reply:
If you don't have a quick instinct on it, I'll just open an issue on vitest and see if they're willing to look at our vitest config since I don't know how else to reproduce it.

Reply:
> . It's not always the tx test run that fails either. I can comment out tx and another one will fail the same way (or I can tell vitest to skip the one test file that's not being loaded and it will still fail on tx with a different file having no tests fail). Very weird.

oh that is really wierd and definitely smells like an internal race condition bug in vitest

Reply:
That was my instinct as well. Will report it and see if they're willing to help.